### DISCUSSION TITLE
2E25136 gets changed to "∞"

### DISCUSSION THREAD

**Discussion by MisterRich** [MasaCMS/MasaCMS#147] [General]
This is a bit of an odd item, and I am trying to figure out what causes it.. If you type 2E25136 into any field, lets say the title.  Something does a math function I am supposing and changes it to infinity in quotes,  Some research tells me that the E is causing it to deal with it as an exponetial function, not text.  I am open to any ideas as to where to look.

**Comment by jochemd** [comment]
Where does it get converted? Is it converted on the client before sending to the backend? In the backend before storage in the database? In the backend when rendered for output?

**Reply by MisterRich** [reply]
I think it might be happening as the form posts, I suspect it to be in the bean validation?  The data looks correct as the form posts, but then it changes to the infinity symbol in quotes before it is saved to the DB.
